test/extensions/filters/network/local_ratelimit/local_ratelimit_fuzz_test.cc

```diff
@@ -0,0 +1,96 @@
+#include "envoy/extensions/filters/network/local_ratelimit/v3/local_rate_limit.pb.h"
+
+#include "common/buffer/buffer_impl.h"
+#include "common/stats/isolated_store_impl.h"
+
+#include "extensions/filters/network/local_ratelimit/local_ratelimit.h"
+
+#include "test/extensions/filters/network/local_ratelimit/local_ratelimit_fuzz.pb.h"
```

Comment:
Don't think you need both this and the compiled code with validation

Reply:
> Don't think you need both this and the compiled code with validation

You are right! Removed.